src/win32/ossec-pre6.conf

```diff
@@ -128,12 +128,10 @@
     <max_eps>200</max_eps>
 
     <!-- Database synchronization settings -->
-    <inventory>
-      <sync_interval>5m</sync_interval>
-      <max_sync_interval>1h</max_sync_interval>
-      <response_timeout>30</response_timeout>
-      <sync_queue_size>16384</sync_queue_size>
-    </inventory>
+    <synchronization>
+      <interval>5m</interval>
```

Comment:
Please add the option `enabled` to the template, in line with the other components